pkgs/development/interpreters/bqn/cbqn/default.nix

```diff
@@ -50,8 +50,8 @@ stdenv.mkDerivation rec {
 
   makeFlags = [
     "CC=${stdenv.cc.targetPrefix}cc"
-  ]
-  ++ lib.optional enableReplxx "REPLXX=1";
+    "REPLXX=${if enableReplxx then "1" else "0"}"
```

Comment:
What effect does adding `env.` prefix do?

Now that you pointed it out, I noted the makefile also allows this as a makefile variable. I think it would be more clean to have it in `buildFlags` without the `env.` prefix, do you agree?

Reply:
> What effect does adding `env.` prefix do?
>

It aggregates in  (Python-esque) dictionary called `env` all the "environment variables" used by the expression.

> Now that you pointed it out, I noted the makefile also allows this as a makefile variable. 

This was the idea all along. I didn't understand why you tried to decouple it from the build phase in the inferior Bash level and exposed it to Nix level.

However, given that you said it helped fix a bug, I took it as granted.

> I think it would be more clean to have it in `buildFlags` without the `env.` prefix, do you agree?

Yes!

Reply:
Thanks for the explanation of `.env`, makes sense now.

> This was the idea all along. I didn't understand why you tried to decouple it from the build phase in the inferior Bash level and exposed it to Nix level.

It was already appended to `makeFlags` and I tried to keep things as they were.

Anyway, now that we both agreed I moved it to `buildFlags` instead.